EpiResponsivePicture/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -82,9 +82,11 @@ private static void ConfigureModule(IServiceCollection services, EpiResponsivePi
         services
             .AddOptions<EpiResponsivePicturesOptions>()
             .Configure(o => {
-                o.ImageResizerCompatibilityEnabled = options?.ImageResizerCompatibilityEnabled ?? false; 
+                o.ImageResizerCompatibilityEnabled = options?.ImageResizerCompatibilityEnabled ?? false;
+                o.AdditionalSegments = options?.AdditionalSegments;
             });
 
+        services.AddScoped<IBlobSegmentsProvider, BlobCustomSegmentsProvider>();
```

Comment:
as far as I can see BlobImageProvider is registered as singleton, so maybe this registration can also be done with Singleton?